apm-agent-core/src/main/java/co/elastic/apm/impl/sampling/ProbabilitySampler.java

```diff
@@ -0,0 +1,30 @@
+package co.elastic.apm.impl.sampling;
+
+import co.elastic.apm.impl.transaction.TransactionId;
+
+public class ProbabilitySampler implements Sampler {
```

Comment:
Maybe some java doc saying what this is?

Reply:
done